core/src/test/java/org/testcontainers/junit/GenericContainerRuleTest.java

```diff
@@ -61,21 +61,21 @@ public static void setupContent() throws FileNotFoundException {
      * Redis
      */
     @ClassRule
-    public static GenericContainer redis = new GenericContainer("redis:3.0.2")
+    public static GenericContainer redis = new GenericContainer("redis:6.0.4")
             .withExposedPorts(REDIS_PORT);
 
     /**
      * RabbitMQ
      */
     @ClassRule
-    public static GenericContainer rabbitMq = new GenericContainer("rabbitmq:3.5.3")
+    public static GenericContainer rabbitMq = new GenericContainer("rabbitmq:3.8.4")
             .withExposedPorts(RABBITMQ_PORT);
 
     /**
      * MongoDB
      */
     @ClassRule
-    public static GenericContainer mongo = new GenericContainer("mongo:3.1.5")
+    public static GenericContainer mongo = new GenericContainer("mongo:4.2.7")
             .withExposedPorts(MONGO_PORT);
```

Comment:
I wonder - should we take the opportunity to clean up this test class a bit?

Way back at the beginning of the project I wanted to test/demonstrate that diverse images could be used - partly to serve as documentation. Now, though, I think these are no longer truly necessary and just add extra image pull time to our build.

Thoughts?

Reply:
I believe this would help reducing the tests time a lot 👍

Reply:
Was thinking the same when touching this class. Maybe the class as a whole is nowadays redundant.

Reply:
Since this is not really related anymore, I will rever this changes and we can do it in a separate cleanup PR.